build/haproxy-entrypoint.sh

```diff
@@ -3,12 +3,18 @@
 set -e
 set -o xtrace
 
+echo "${CLUSTER_TYPE}" >/tmp/cluster_type
+
 if [ "$1" = 'haproxy' ]; then
 	if [ ! -f '/etc/haproxy/mysql/haproxy.cfg' ]; then
 		cp /opt/percona/haproxy.cfg /etc/haproxy/mysql
 	fi
 
 	haproxy_opt='-W -db -f /opt/percona/haproxy-global.cfg -f /etc/haproxy/mysql/haproxy.cfg -p /etc/haproxy/mysql/haproxy.pid -S /etc/haproxy/mysql/haproxy-main.sock'
+
```

Comment:
@egegunes please use the same approach for custom conf as we have for PXC operator
https://github.com/percona/percona-docker/blob/main/haproxy/dockerdir/entrypoint.sh#L17C5-L28

In case of custom config we need to replace whole conf

Reply:
done